modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -120,21 +94,34 @@ xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read
 * Xen
 * KVM
 
-=== Desktop
+=== Mobile OS
```

Comment:
If 10.3 branch says "Mobile OS" here, then IMO the heading is wrong in 10.3, and needs to be fixed there.

Reply:
I just corrected it in this branch and will correct it in 10.3